api/docs/release.dox

```diff
@@ -198,6 +198,7 @@ Further non-compatibility-affecting changes include:
    sec_drcachesim_ops).
  - Added a last-level cache miss recording feature to drcachesim.
  - Added a delayed tracing feature to drcachesim.
+ - Added an option to specify cache warmup by loaded fraction of the last level cache.
```

Comment:
Should mention drcachesim as this is not an option to DR (see all the other drcachesim changes above): s/option/option to drcachesim/